core/domain/platform_parameter_list.py

```diff
@@ -30,38 +30,38 @@
 # to PascalCase and its values to UPPER_CASE. Because we want to be consistent
 # throughout the codebase according to the coding style guide.
 # https://github.com/oppia/oppia/wiki/Coding-style-guide
-class PARAM_NAMES(enum.Enum): # pylint: disable=invalid-name
+class ParamNames(enum.Enum):
     """Enum for parameter names."""
 
-    dummy_feature = 'dummy_feature' # pylint: disable=invalid-name
-    dummy_parameter = 'dummy_parameter' # pylint: disable=invalid-name
+    DUMMY_FEATURE = 'dummy_feature'
+    DUMMY_PARAMETER = 'dummy_parameter'
 
-    end_chapter_celebration = 'end_chapter_celebration' # pylint: disable=invalid-name
-    checkpoint_celebration = 'checkpoint_celebration' # pylint: disable=invalid-name
+    END_CHAPTER_CELEBRATION = 'end_chapter_celebration'
+    CHECKPOINT_CELEBRATION = 'checkpoint_celebration'
 
 
 # Platform parameters should all be defined below.
 
 Registry.create_feature_flag(
-    PARAM_NAMES.dummy_feature,
-    'This is a dummy feature flag.',
+    ParamNames.DUMMY_FEATURE,
+    'This is a dummy feature FLAG.',
```

Comment:
Ditto here--I wouldn't expect the actual value to change.

Reply:
Done